rpcs3/Input/keyboard_pad_handler.cpp

```diff
@@ -638,6 +638,18 @@ std::string keyboard_pad_handler::GetKeyName(const QKeyEvent* keyEvent)
 		return "Meta";
 	case Qt::Key_NumLock:
 		return sstr(QKeySequence(keyEvent->key()).toString(QKeySequence::NativeText));
+#ifdef __APPLE__
+	// QKeySequence doesn't work properly for the arrow keys on macOS
+	// Also change the "keypad" arrows to regular arrows
```

Comment:
I think this should be changed to something more understandable
```suggestion
	// Using QKeySequence, the regular arrow keys are named "Num←", "Num↑", "Num→" and "Num↓" on macOS.
	// Let's rename those keys to make it less confusing for the user.
```